pkg/proxy/types.go

```diff
@@ -48,23 +48,26 @@ func (spn ServicePortName) String() string {
 type ServicePort interface {
 	// String returns service string.  An example format can be: `IP:Port/Protocol`.
 	String() string
-	// ClusterIP returns service cluster IP.
-	ClusterIP() string
-	// Protocol returns service protocol.
-	Protocol() api.Protocol
-	// HealthCheckNodePort returns service health check node port if present.  If return 0, it means not present.
-	HealthCheckNodePort() int
+	// ClusterIPString returns service cluster IP in string format.
+	ClusterIPString() string
```

Comment:
Ref https://github.com/kubernetes/kubernetes/pull/56880#discussion_r169744581, Pavithra pointed out this is returning a string, instead of returning the ClusterIP field, which is in net.IP. Renaming makes this purpose clear.